contracts/ENSRegistry.sol

```diff
@@ -100,4 +127,35 @@ contract ENSRegistry is ENS {
   function ttl(bytes32 _node) external view override returns (uint64) {
     return records[_node].ttl;
   }
+
+  /**
+    * @dev Returns whether a record has been imported to the registry.
+    * @param _node The specified node.
+    * @return Bool if record exists
+    */
+  function recordExists(bytes32 _node) public override view returns (bool) {
+      return records[_node].owner != address(0x0);
+  }
+
+  /**
+    * @dev Query if an address is an authorized operator for another address.
+    * @param _owner The address that owns the records.
+    * @param _operator The address that acts on behalf of the owner.
+    * @return True if `operator` is an approved operator for `owner`, false otherwise.
+    */
+  function isApprovedForAll(address _owner, address _operator) external override view returns (bool) {
```

Comment:
this one is the one I need feedback. So to make the BaseRegistrar work without change it, we need to have the isApprovedForAll interace. I can do two things: because our ENS registry is different, and we don't use operators, I just return false. Or I implement our ENS registry same as it is found in ensdomains. Please let me know